src/components/Menu/Menu.module.css

```diff
@@ -17,34 +17,35 @@
  */
 
 .menu {
+  --mp: calc(var(--spacing-padding-md, 12) * 1px);
```

Comment:
What does "mp" stand for? Let's use a clearer name before merging into the main branch.

Reply:
significa "Menu di riempimento". è stato compresso per evitare di riscrivere caratteri lunghi nella stenografia dei campi di riempimento. but if you don't like it, let's write it extended 😄

Reply:
i change local var names